woocommerce/woocommerce-ios
# [Mobile Payments] Manage Payment Gateway from Settings

Closes: #6992 


### Description


This adds a new row to In-Person Payments settings to manage the payment gateways when there is more than one, including some slight redesign of the settings screen to group items better.

When you tap on that row, the UX is a bit unexpected, but we decided it was a good compromised of effort vs how often we expect this to be needed. Instead of pushing a new selection screen, tapping on Payment options clears the preferred gateway and restarts the onboarding checks, which will cause the selection screen to appear.

Note that this setting is not yet respected when taking payments, #7086 will take care of that

### Testing instructions


Go to Settings > In-Person Payments 
1. On a store with only WCPay or Stripe installed, there should be no new row 
2. On a store with both extensions, the row should appear. Tapping on it clears the selection and shows the selection screen
3. Any selection is persisted and shows up the next time you visit the settings 

### Screenshots


One gateway only | WCPay selected | Stripe selected
-|-|-
![Screen Shot 2022-06-14 at 16 21 14](https://user-images.githubusercontent.com/8739/173600854-091ad873-1ef5-4d37-b717-a3aeeded45b0.png)|![Screen Shot 2022-06-14 at 16 03 47](https://user-images.githubusercontent.com/8739/173596692-951526e5-e1c5-4191-b700-b5abef642fed.png)|![Screen Shot 2022-06-14 at 16 03 57](https://user-images.githubusercontent.com/8739/173596700-4ecbf68d-1db2-4e83-b78f-ffcac73db258.png)


---
